core/src/main/scala/org/genivi/sota/core/data/Campaign.scala

```diff
@@ -10,17 +10,19 @@ import eu.timepit.refined.string._
 import io.circe._
 import java.time.Instant
 import java.util.UUID
+
 import org.genivi.sota.core.SotaCoreErrors
 import org.genivi.sota.data.{Namespace, PackageId, Uuid}
 import slick.driver.MySQLDriver.api._
-
 import Campaign._
+import com.advancedtelematic.libats.codecs.{CirceEnum, SlickEnum}
+import org.genivi.sota.core.data.CampaignStatus.Status
 
 case class Campaign (meta: CampaignMeta, packageId: Option[PackageId], groups: Seq[CampaignGroup]) {
   def canLaunch(): Throwable Xor Unit = {
-    if (!meta.packageUuid.isDefined || groups.size < 1) {
+    if (meta.packageUuid.isEmpty || groups.size < 1) {
       Xor.Left(SotaCoreErrors.CantLaunchCampaign)
-    } else if (meta.launched) {
+    } else if (meta.status == CampaignStatus.Active) {
```

Comment:
Well you changed your previous comment so of course my comment doesn't make any sense anymore. 

So I think we should only be allowed to launch if we are in `Draft` mode so if we are not in `Draft` we should give the error.